usl_pipeline/usl_lib/usl_lib/readers/config_readers.py

```diff
@@ -21,3 +21,22 @@ def read_rainfall_amounts(rain_fd: TextIO) -> Sequence[float]:
             continue
         entries.append(float(rainfall_line.group(1)))
     return entries
+
+
+def read_key_value_pairs(txt_fd: TextIO) -> dict:
```

Comment:
If we're creating the config file and deciding its format, I think we should pick an existing format with existing readers & writers rather than build our own?

What you've got looks pretty similar to an ini file
https://docs.python.org/3/library/configparser.html

I belive you can create an ini file without the [section] parts, which looks like exactly what you're supporting here.

so I'd just go with that existing ini support, or something like JSON or YAML

Reply:
Gotcha - so the contents can stay the same but use `.ini` file extension and replace the custom reader with the configparser lib? Or could we user configparser on .txt files as well?

Reply:
the parser won't care if the extension of the file is .ini or .txt